version.properties

```diff
@@ -0,0 +1,4 @@
+#Version of the produced binaries. This file is intended to be checked-in.
+#It will be automatically bumped by release automation.
+version=1.0.1
+previousVersion=1.0.0
```

Comment:
Should the first version be 1.0.0?

Reply:
Let me verify If I can rollback this. This actually automatically changed by the system.

Reply:
I leave this open till next update. I will try and do a local test